### PR TITLE
Some Backend Admin fixes and updates

### PR DESCRIPTION
**Pull request by BlackDex**
- Updated datatables
- Added a `X-Robots-Tags` header to prevent indexing
- Modified some layout settings
- Added Websocket check to diagnostics
- Added Security Header checks to diagnostics
- Added Error page response checks to diagnostics
- Modifed support string layout a bit

### CODE REVIEW

File: src/static/scripts/admin_diagnostics.js [hunk 1]
```diff
@@ -199,6 +219,162 @@ function checkDns(dns_resolved) {
...
+        return { headers: response.headers, status: response.status, text: await response.text() };
+    } catch (error) {
+        console.error(`Error fetching ${url}: ${error}`);
+        return { error };
+    }
+}
+
+function checkSecurityHeaders(headers, omit) {
+    let securityHeaders = {
+        "x-frame-options": ["SAMEORIGIN"],
```
Comment (dani-garcia) [inline src/static/scripts/admin_diagnostics.js:234]: Probably a good idea to add a comment in `util.rs` where we declare these headers mentioning that if they are modified they also need to be updated here.

File: src/static/scripts/admin_diagnostics.js [hunk 2]
```diff
@@ -199,6 +219,162 @@ function checkDns(dns_resolved) {
...
+        }
+    }
+    return messages;
+}
+
+async function checkHttpResponse() {
+    const [apiConfig, webauthnConnector, notFound, notFoundApi, badRequest, unauthorized, forbidden] = await Promise.all([
+        fetchCheckUrl(`${BASE_URL}/api/config`),
+        fetchCheckUrl(`${BASE_URL}/webauthn-connector.html`),
+        fetchCheckUrl(`${BASE_URL}/admin/does-not-exists`),
```
Comment (dani-garcia) [inline src/static/scripts/admin_diagnostics.js:279]: Doesn't really matter, but typo
```suggestion
        fetchCheckUrl(`${BASE_URL}/admin/does-not-exist`),
```

**Review by dani-garcia — COMMENTED** [review]
LGTM, just some small comments

File: src/static/scripts/admin_diagnostics.js (hunk 1 above)
Comment (BlackDex) [reply src/static/scripts/admin_diagnostics.js:234]: I mend to do that, but forgot of course.

File: src/static/scripts/admin_diagnostics.js (hunk 2 above)
Comment (BlackDex) [reply src/static/scripts/admin_diagnostics.js:279]: Or wasn't it ;)

**Pull request merged**